docs/nodes/nodes-library/core-nodes/Compression/README.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /nodes/n8n-nodes-base.compression
+description: Learn how to use the Compression node in n8n
+---
+
+# Compression
+
+The Compression node is useful to compress and decompress files. You can either use the `gzip` or `zip` file format.
+
+
+## Node Reference
+
+- Operations
+	- Compress
+	- Decompress
+
+- Options
+    - ***Binary Property*** field: This field allows you to specify the name of the binary property
+    - ***Output Prefix*** field: This field allows you to specify a prefix for the name of the file gets generated
+    - ***Output Format*** field: This field allows you to select an output format. This field is displayed when 'Compress' gets selected from the ***Operation*** dropdown list. The following are the available formats:
+        - gzip
+        - zip
+    - ***File Name*** field: This field allows you to specify a file name. This field is displayed when 'zip' gets selected from the ***Output Format*** dropdown list.
+
+## Example Usage
+
+This workflow allows you to compress binary files to zip format. You can also find the [workflow](https://n8n.io/workflows/908) on n8n.io. This example usage workflow would use the following nodes.
+- [Start](../../core-nodes/Start/README.md)
+- [HTTP Request](../../core-nodes/HTTPRequest/README.md)
+- [Compression]()
+
```

Comment:
We should add a write binary file node or something similar after this so that they can actually interact with the compressed files after the workflow.

Reply:
I was thinking the same but for the cloud users, the Write Binary File node won't be an ideal example. How about we use Dropbox or something similar where the user can upload the file?

Reply:
Sounds good